src/config/configuration.ts

```diff
@@ -13,7 +13,7 @@ export interface AppConfig {
   port?: number;
 }
 
-export default registerAs('app', () => {
+export default registerAs('', () => {
```

Comment:
Is an empty string okay here?

Reply:
`app` is for namespace and managing multiple configurations.  
if we have only one configuration and namespace we can remove `registerAs`  function too.

Reply:
`app` is for namespace and managing multiple configurations.  
if we have only one configuration and namespace we can remove `registerAs`  function too.

Reply:
> Is an empty string okay here?

yes it's ok